app/scripts/modules/openstack/loadBalancer/configure/wizard/upsert.controller.js

```diff
@@ -133,6 +137,19 @@ module.exports = angular.module('spinnaker.loadBalancer.openstack.create.control
       $scope.existingLoadBalancerNames = _.flatten(_.map(allLoadBalancerNames[$scope.loadBalancer.account || ''] || []));
     }
 
+    function updateSecurityGroups() {
+      var account = _.get($scope, ['loadBalancer', 'account']);
+      var region = _.get($scope, ['loadBalancer', 'region']);
+      if (!account || !region) {
+        $scope.allSecurityGroups = [];
+      }
+      var securityGroups = securityGroupReader.getAllSecurityGroups()
```

Comment:
Looks like `securityGroups` isn't being used here (it's also shadowing `securityGroups` in the `.then`).